presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloMetadata.java

```diff
@@ -331,20 +331,14 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
     }
 
     @Override
-    public List<SchemaTableName> listTables(ConnectorSession session, String schemaNameOrNull)
+    public List<SchemaTableName> listTables(ConnectorSession session, Optional<String> schemaName)
     {
-        Set<String> schemaNames;
-        if (schemaNameOrNull != null) {
-            schemaNames = ImmutableSet.of(schemaNameOrNull);
-        }
-        else {
-            schemaNames = client.getSchemaNames();
-        }
+        Set<String> schemaNames = schemaName.<Set<String>>map(ImmutableSet::of).orElseGet(client::getSchemaNames);
```

Comment:
is `<Set<String>>` required?
put `\n` before every `.`, especially that `client::getSchemaNames` is so important

Reply:
`<Set<String>>` is needed in this construct, I could use something else (maybe plain `if`) if you do not like it